app/experimenter/experiments/tests/api/v5/test_serializers/test_nimbus_experiment_serializer.py

```diff
@@ -941,6 +971,65 @@ def test_can_request_review_from_preview(self):
         self.assertEqual(experiment.status, NimbusExperiment.Status.DRAFT)
         self.assertEqual(experiment.publish_status, NimbusExperiment.PublishStatus.REVIEW)
 
+    def test_rollout_can_request_review_from_live(self):
```

Comment:
It's sort of implicit here that it's going from IDLE -> REVIEW but maybe explicitly stating that in the test name would make that clearer

Reply:
:salute: